scripts/build-json/build-json.js

```diff
@@ -2,41 +2,51 @@ const fs = require('fs');
 const path = require('path');
 
 const buildPage = require('./build-page-json');
+const { ROOT } = require('./constants');
 
-function walk(directory, filepaths) {
+
+function findItems(directory, searchPaths, filepaths = []) {
```

Comment:
I messed with this function a little because it hurt my soul to see that we're first looping over the file system and once we've done that we loop over all the files again just to filter them. 
And the way it was done before, if `searchPaths` was an empty array it would always loop over all of them for no reason because `!searchPaths.length` would always be true.